### PR TITLE
br: modify metric name in alert rule and add alert examples

### PR DESCRIPTION
**Pull request by WangLe1321**
Signed-off-by: WangLe1321 <wangle1321@163.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by fubinzh — APPROVED** [review]
LGTM

File: br/pitr-monitoring-and-alert.md [hunk 1]
```diff
@@ -49,38 +49,57 @@ PiTR 支持使用 [Prometheus](https://prometheus.io/) 采集监控指标，目
 
 目前 PiTR 还未内置告警项，以下告警项为推荐的配置。
 
+告警规则配置可以参考下面的步骤：
+1. 在 Prometheus 所在节点创建告警规则配置文件（如 pitr.rules.yml），参考 [Prometheus 文档](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/) 和下列推荐告警项及配置样例填写告警规则；
+2. 在 Prometheus 配置文件中的 `rule_files` 字段填入告警规则文件路径；
+3. 通过向 Prometheus 进程发送 `SIGHUP` 信号（`kill -HUP pid`）或向 `http://prometheus-addr/-/reload` 发送 HTTP POST 请求（WEB 请求的方式需要启动 Prometheus 时指定 `--web.enable-lifecycle`）。
```
Comment (TomShawn) [inline br/pitr-monitoring-and-alert.md:55]:
```suggestion
目前 PiTR 无内置告警项。本节介绍如何在 PiTR 中配置告警项，以及推荐的告警项规则。

可以参考以下步骤在 PiTR 中配置告警项：

1. 在 Prometheus 所在节点上创建告警规则的配置文件（例如 `pitr.rules.yml`），参考 [Prometheus 文档](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/) 和下列推荐的告警项及配置样例填写告警规则。
2. 在 Prometheus 配置文件中的 `rule_files` 字段填入告警规则文件的路径。
3. 通过向 Prometheus 进程发送 `SIGHUP` 信号（`kill -HUP pid`）或向 `http://prometheus-addr/-/reload` 发送 HTTP POST 请求（使用 WEB 请求的方式前需要在启动 Prometheus 时指定 `--web.enable-lifecycle`）。

以下为推荐的告警项配置：
```

File: br/pitr-monitoring-and-alert.md [hunk 2]
```diff
@@ -49,38 +49,57 @@ PiTR 支持使用 [Prometheus](https://prometheus.io/) 采集监控指标，目
...
+
 ### LogBackupRunningRPOMoreThan10m
 
-- 表达式：max(time() - tikv_stream_store_checkpoint_ts / 262144000) by (task) / 60 > 10 and max(tikv_stream_store_checkpoint_ts) by (task) > 0 and max(tikv_log_backup_task_status) by (task) == 0
+- 表达式：max(time() - tikv_log_backup_store_checkpoint_ts / 262144000) by (task) / 60 > 10 and max(tikv_log_backup_store_checkpoint_ts) by (task) > 0 and max(tikv_log_backup_task_status) by (task) == 0
 - 告警级别：warning
-- 说明：日志数据超过 10 分钟未持久化到存储中，该配置项主要用于提醒，大部分情况下，不会影响日志备份
+- 说明：日志数据超过 10 分钟未持久化到存储中，该配置项主要用于提醒，大部分情况下，不会影响日志备份。
+
+**Prometheus 配置样例：**
```
Comment (TomShawn) [inline br/pitr-monitoring-and-alert.md:63]:
```suggestion
Prometheus 中的配置样例如下：

```

File: br/pitr-monitoring-and-alert.md [hunk 3]
```diff
@@ -49,38 +49,57 @@ PiTR 支持使用 [Prometheus](https://prometheus.io/) 采集监控指标，目
...
+      severity: warning
+    annotations:
+      summary: RPO of log backup is high
+      message: RPO of the log backup task {{ $labels.task }} is more than 10m
+```
 
 ### LogBackupRunningRPOMoreThan30m
 
-- 表达式：max(time() - tikv_stream_store_checkpoint_ts / 262144000) by (task) / 60 > 30 and max(tikv_stream_store_checkpoint_ts) by (task) > 0 and max(tikv_log_backup_task_status) by (task) == 0
+- 表达式：max(time() - tikv_log_backup_store_checkpoint_ts / 262144000) by (task) / 60 > 30 and max(tikv_log_backup_store_checkpoint_ts) by (task) > 0 and max(tikv_log_backup_task_status) by (task) == 0
```
Comment (TomShawn) [inline br/pitr-monitoring-and-alert.md:79]: 建议所有的表达式都用反引号包裹起来
```suggestion
- 表达式：`max(time() - tikv_log_backup_store_checkpoint_ts / 262144000) by (task) / 60 > 30 and max(tikv_log_backup_store_checkpoint_ts) by (task) > 0 and max(tikv_log_backup_task_status) by (task) == 0`
```

File: br/pitr-monitoring-and-alert.md [hunk 4]
```diff
@@ -47,40 +47,63 @@ PiTR 支持使用 [Prometheus](https://prometheus.io/) 采集监控指标，目
...
 ## 告警配置
 
-目前 PiTR 还未内置告警项，以下告警项为推荐的配置。
+目前 PiTR 还未内置告警项，本节介绍如何在 PiTR 中配置告警项，以及推荐的告警项规则。
+
+告警规则配置可以参考下面的步骤：
+
+1. 在 Prometheus 所在节点创建告警规则配置文件（如 pitr.rules.yml），参考 [Prometheus 文档](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/) 和下列推荐告警项及配置样例填写告警规则；
+2. 在 Prometheus 配置文件中的 `rule_files` 字段填入告警规则文件路径；
+3. 通过向 Prometheus 进程发送 `SIGHUP` 信号（`kill -HUP pid`）或向 `http://prometheus-addr/-/reload` 发送 HTTP POST 请求（WEB 请求的方式需要启动 Prometheus 时指定 `--web.enable-lifecycle`）。
```
Comment (TomShawn) [inline br/pitr-monitoring-and-alert.md:56]:
```suggestion
1. 在 Prometheus 所在节点上创建告警规则的配置文件（例如 `pitr.rules.yml`），参考 [Prometheus 文档](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/) 和下列推荐的告警项及配置样例填写告警规则。
2. 在 Prometheus 配置文件中的 `rule_files` 字段填入告警规则文件的路径。
3. 通过向 Prometheus 进程发送 `SIGHUP` 信号（`kill -HUP pid`）或向 `http://prometheus-addr/-/reload` 发送 HTTP POST 请求（使用 WEB 请求的方式前需要在启动 Prometheus 时指定 `--web.enable-lifecycle` 参数）。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by IANTHEREAL — APPROVED** [review]

**Pull request merged**